Shared/Samples/Take screenshot/TakeScreenshotView.swift

```diff
@@ -0,0 +1,118 @@
+// Copyright 2025 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+struct TakeScreenshotView: View {
+    /// The current draw status of the map.
+    @State private var currentDrawStatus: DrawStatus = .inProgress
+    /// The screenshot to export.
+    @State private var screenshotToExport: Screenshot?
```

Comment:
Should this simply be named `screenshot`? The screenshot is displayed and allows for export, so naming it `ToExport` seems overly verbose.